argoverse/data_loading/object_label_record.py

```diff
@@ -191,6 +203,25 @@ def draw_rect(selected_corners: np.array, color: Tuple[int, int, int]) -> None:
         # Draw rear (last 4 corners) in red
         draw_rect(corners[4:], colors[1][::-1])
 
+        # grab the top vertices
+        center_top = np.mean(corners[[0,1,4,5]], axis=0)
+        uv_ct, _, _, _ = proj_cam_to_uv(center_top.reshape(1, 3), camera_config)
+        uv_ct = uv_ct.squeeze()
+
+        if np.linalg.norm(center_top) < MAX_RANGE_THRESH_PLOT_CATEGORY and uv_ct[0] >= 0 and uv_ct[1] >= 0:
```

Comment:
Potentially make this a helper function which returns a boolean w/ a descriptive name?

Reply:
that's a good idea, thanks